releases/release-8.5.3.md

```diff
@@ -0,0 +1,149 @@
+---
+title: TiDB 8.5.3 Release Notes
+summary: 了解 TiDB 8.5.3 版本的兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 8.5.3 Release Notes
+
+发版日期：2025 年 x 月 x 日
+
+TiDB 版本：8.5.3
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v8.5/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v8.5/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v8.5.3#version-list)
+
+## 兼容性变更
+
+- note [#issue](https://github.com/pingcap/${repo-name}/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
```

Comment:
@Defined2014 Since this PR (https://github.com/pingcap/docs-cn/pull/20556) introduces a compatibility change, we should include a release note to help users understand the impact. Please add one following the pattern:

```
- note [#issue](https://github.com/pingcap/${repo-name}/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
```